code/game/machinery/computer/camera_console.dm

```diff
@@ -160,55 +134,18 @@
 		if(!selected_camera)
 			return TRUE
 
-		update_active_camera_screen()
+		SEND_SIGNAL(src, COMSIG_CAMERA_SET_TARGET, selected_camera, selected_camera.view_range, selected_camera.view_range)
```

Comment:
It is currently moot to assign `w` and `h` because `/datum/component/camera_manager/proc/set_camera` merely sets those into `target_width` and `target_height` which isn't used for anything.

Reply:
Reworked this so that it uses the target_width and target_height being passed in via signal. It slightly reduces decoupling between the component and the camera.